specifyweb/frontend/js_src/lib/components/SchemaConfig/index.tsx

```diff
@@ -109,7 +109,7 @@ export function SchemaConfigMain(): JSX.Element {
         languages={schemaData.languages}
         onSave={canSave ? handleSave : undefined}
       />
-      <div className="flex flex-1 flex-col gap-4 overflow-hidden sm:flex-row">
+      <div className="flex flex-1 flex-col gap-4 overflow-y-auto sm:flex-row">
```

Comment:
I belive `overflow-hidden` was there for a reason. Can you please test to make sure it is not needed? What is the behavior before and after the change?